src/components/AdequacyDoughnut/AdequacyDoughnut.tsx

```diff
@@ -17,19 +19,29 @@ type Props = StoreProps & {
  */
 (Chart as any).defaults.global.legend.labels.usePointStyle = true
 
+
```

Comment:
This is what is breaking your linter. It doesn't like these additional blank lines here.